### PR TITLE
[13.0][MIG] purchase_force_invoiced

### PR DESCRIPTION
**Pull request by kongrattapong**
Ready for review

### CODE REVIEW

**Comment by kittiu** [comment]
Functional and technical tested,
LGTM :+1:

**Review by kittiu — APPROVED** [review]

File: purchase_force_invoiced/__manifest__.py [hunk 1]
```diff
@@ -0,0 +1,17 @@
+# Copyright 2019 Eficent Business and IT Consulting Services S.L.
+# Copyright 2019 Aleph Objects, Inc.
+# License AGPL-3.0 or later (https://www.gnu.org/licenses/agpl.html)
+
+{
+    "name": "Purchase Force Invoiced",
+    "summary": "Allows to force the billing status of the purchase order to "
+    '"Invoiced"',
+    "version": "13.0.1.0.0",
+    "author": "Eficent," "Odoo Community Association (OCA)",
```
Comment (HviorForgeFlow) [inline purchase_force_invoiced/__manifest__.py]:
```suggestion
    "author": "Eficent, Odoo Community Association (OCA)",
```

**Review by HviorForgeFlow — CHANGES_REQUESTED** [review]
In addition, can you change Eficent -> Forgeflow? Thanks!

**Comment by kittiu** [comment]
> In addition, can you change Eficent -> Forgeflow? Thanks!

Edited. Thanks!

**Review by HviorForgeFlow — COMMENTED** [review]
Can you rebase this branch? Thanks!

**Comment by kittiu** [comment]
Rebased

**Review by HviorForgeFlow — APPROVED** [review]

File: purchase_force_invoiced/view/purchase_view.xml [hunk 2]
```diff
@@ -1,19 +1,15 @@
-<?xml version="1.0" encoding="utf-8"?>
```
Comment (HviorForgeFlow) [inline purchase_force_invoiced/view/purchase_view.xml]: XML header is missing

**Review by HviorForgeFlow — CHANGES_REQUESTED** [review]

**Review by HviorForgeFlow — APPROVED** [review]

**Pull request merged**